docs/docs/get-started/quick-tour.md

```diff
@@ -9,6 +9,13 @@
 
     If you don't have an OpenAI API key, please get one [here](https://platform.openai.com/docs/quickstart/developer-quickstart)
 
+    create or append the following to the .env file at the root directory of the project
```

Comment:
As far as I can see, this should not work actually. For Linux users, you would need to add `export VAR="your_key"` to your .bashrc file

I am working on a FAQ/support/troubleshooting guide so perhaps I can include a `How do I set up my environment variables?`question and provide an answer for multiple environments here. Then in the quick tour I can just link to that. What do you think?